AUTHORS

```diff
@@ -1,6 +1,9 @@
-# This file lists all individuals having contributed content to the repository.
+# This file lists all individuals having
+contributed content to the repository.
 
-
-Jennifer Huang <133@holbertonschool.com>
-Alexa Orrico <210@holbertonschool.com>
-Joann Vuong <130@holbertonschool.com> 
+Alexa Orrico <alexaorrico@gmail.com>
```

Comment:
Seems like the script you wrote doubled up some people

Reply:
It's not super important in relation to the code working, just something that seemed a little weird

Reply:
Ah! not my script, one from a previous airbnb clone repo. We can ditch it and manually edit AUTHORS. Good catch!